builder

```diff
@@ -38,6 +38,7 @@ readonly kernel_source_dir="${kernel_dir}/source"
 mkdir -p "${kernel_source_dir}"
 readonly kernel_bzimage="${kernel_source_dir}/arch/x86/boot/bzImage"
 readonly kernel_reboot_patch_url="https://raw.githubusercontent.com/coreos/rkt/v1.22.0/stage1/usr_from_kvm/kernel/patches/0001-reboot.patch"
+readonly kernel_api_header_dir="/lib/modules/${kernel_version}${kernel_version_suffix}"
 readonly kernel_header_dir="/lib/modules/${kernel_version}${kernel_version_suffix}/source"
```

Comment:
`kernel_source_dir` is already used further above. I can change the names, but IMHO it's not super important / we can do it later. (I have also been thinking if we should put the headers into e.g. `/kernel-header`  instead.)